xAPI-Communication.md

```diff
@@ -1667,246 +1667,18 @@ described in [Appendix A: Converting Statements to 1.0.0](#Appendix3A).
-* <a name="4.1.s3.b2"></a>If this form of authentication is used to record Statements and no authority is specified, the LRS 
-SHOULD record the authority as a Group consisting of an Agent representing the registered application, and an Agent 
-representing the known user.
-
-###### <a name="4.1.s4"></a>Application registered + user unknown Process and Requirements
-
-**Process:** 
-The LRS honors requests that are signed using OAuth with the registered application's credentials and with an empty token 
-and token secret.
-
-**Requirements:**
-* <a name="4.1.s4.b1"></a>If this form of authentication is used to record Statements, the LRS SHOULD record the authority 
-as the Agent representing the registered application.
-
-###### <a name="4.1.s5"></a>Application Not Registered + Known User Process and Requirements
-
-**Process:**
-The Learning Record Provider uses a consumer secret consisting of an empty string to call the Temporary Credential 
-Request endpoint specifying the "consumer_name" and other usual parameters.  The "consumer_name" contains a string 
-representing the application requesting access. 
-
-The Learning Record Provider then sends the user's browser to the Resource Owner Authorization using the temporary 
-credentials obtained from the LRS. The Resource Owner Authorization presents a page displaying the "consumer_name" 
-plus a warning that the identity of the application requesting authorization cannot be verified.
-
-Otherwise the process follows the standard OAuth workflow. 
-
-**Requirements:**
-* <a name="4.1.s5.b1"></a>If this form of authentication is used to record Statements, the LRS MUST record an authority 
-that includes both that application and the authenticating user, as a Group, since OAuth specifies an application.
-
-###### <a name="4.1.s6"></a>No Application + Known User Process and Requirements
-**Process:**
-Use a username/password combination provided by the LRS for use by the known user.
-
-**Requirements:**
-* <a name="4.1.s6.b1"></a>If this form of authentication is used to record Statements, the LRS SHOULD 
-record the authority as the Agent representing the known user.
-
-###### <a name="4.1.s7"></a>No Authorization Process and Requirements
-
-* <a name="4.1.s7.b1"></a>Requests MUST include headers for HTTP Basic Authentication based on a username and password 
-containing zero or more space characters. 
-* <a name="4.1.s7.b2"></a>Requests SHOULD* include headers for HTTP Basic Authentication based on a username and password 
-each consisting of an empty string. In this case the HTTP Basic Authentication header will be `Basic ` followed by a base64 
-encoded version of the string `:`.  This results in the string `Basic Og==`.
-
-This is in order to distinguish an explicitly unauthenticated request from a request that needs to be given a 
-HTTP Basic Authentication challenge.
-
-<a name="oauthscope"></a> 
-
-### <a name="4.2">4.2</a> OAuth 1.0 Authorization Scope
```

Comment:
we could try to keep a more general concept of "scope" but that seems like more trouble than it's worth, and of limited value for interoperability.

Reply:
I'd rather see that covered separately as best practice guidelines and see how that works out before standardizing scopes. 

I'm also not sure if standardizing scopes is needed for interoperability. 

So +1 for removing scopes.